pagerduty/resource_pagerduty_event_orchestration_path_router.go

```diff
@@ -63,9 +63,29 @@ func resourcePagerDutyEventOrchestrationPathRouter() *schema.Resource {
 										MaxItems: 1, // there can only be one action for router
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
+												"dynamic_route_to": {
+													Type:     schema.TypeList,
+													Optional: true,
+													Elem: &schema.Resource{
+														Schema: map[string]*schema.Schema{
+															"lookup_by": {
+																Type:     schema.TypeString,
+																Required: true,
+															},
+															"regex": {
+																Type:     schema.TypeString,
+																Required: true,
+															},
+															"source": {
+																Type:     schema.TypeString,
+																Required: true,
+															},
+														},
+													},
+												},
 												"route_to": {
 													Type:     schema.TypeString,
-													Required: true,
+													Optional: true,
```

Comment:
**Annotation:** We can't have either `dynamic_route_to` or `route_to` required because they are mutually exclusive and a rule will have either one. Hopefully in the subsequent PR we can add a check for a rule to have  one or the other.